### ISSUE TITLE
Security Vulnerbility

### ISSUE THREAD

**Issue by Hypnos3** [thibauts/node-castv2#54]
I have get an [issue ](https://github.com/Hypnos3/node-red-contrib-cast/issues/32) from @rgerrans. 
This issue is depending on the used protobuf version:

![title](https://user-images.githubusercontent.com/18623772/50674495-1f8e8d80-0fa4-11e9-8fd8-89a706317bc5.png)

Is it possible to update the used version?

**Comment by lacker** [comment]
npm is starting to warn about the dependency here, too. Although since it's been a couple years since this repo has been updated, I won't hold my breath.

**Comment by rgerrans** [comment]
> npm is starting to warn about the dependency here, too. Although since it's been a couple years since this repo has been updated, I won't hold my breath.

I ended up switching to https://github.com/Hypnos3/node-red-contrib-cast

**Comment by dsteinman** [comment]
Upgrading the dependency to 5.0.3 works, at least I didn't encounter anything that with this change broke yet.

```
"dependencies": {
    "debug": "^2.2.0",
    "protobufjs": "^5.0.3"
  },
```

**Comment by Saiv46** [comment]
Any progress here? `npm audit` still reporting this vulnerable package.

**Comment by feross** [comment]
Hi everyone, I have a pull request that fixes this issue but I don't have a Chromecast handy at the moment to test it out. Can someone confirm that the code in this pull request still works correctly?

https://github.com/thibauts/node-castv2/pull/56

**Comment by Hypnos3** [comment]
Thank you for the change. I can test not before end of next week.

**Comment by feross** [comment]
Fixed in castv2@0.1.10.
